src/containers/Sidebar/sidebar.scss

```diff
@@ -124,8 +129,9 @@ $sidebar-transition-timing-function: cubic-bezier(0.6, 0.6, 0, 1);
 
 	transition: all 0.2s linear; // This is speed of sidebar collapsing
 
+
 	& #app-sidebar {
-		width: 3rem;
+		width: 3.5rem;
```

Comment:
I assume `3.5rem` may be too much, it diminish space for the main screen (you can see it from the attached image, that the edge of the sidebar is touching the card with Credentials